src/DataGrid.tsx

```diff
@@ -1033,6 +1033,7 @@ function DataGrid<R, SR, K extends Key>(
             viewportColumns={rowColumns}
             childRows={row.childRows}
             rowIdx={rowIdx}
+            lastFrozenColumnIndex={lastFrozenColumnIndex}
```

Comment:
Is it possible to set the selected row style using the extra div instead?
https://github.com/adazzle/react-data-grid/blob/3847e2f1d15f5766f05c0945f2e00b49672d1889/src/DataGrid.tsx#L1150

Reply:
Not sure, would still need something to set a border for the first sticky column